docs/docsource/operation/XDM_GUI/bucket_operations/update_a_replication_workflow.rst

```diff
@@ -40,6 +40,7 @@ Where ``configuration`` is a JSON object including the following syntax:
            }
          }
        ]
+     "Role": "arn:aws:iam::root:role/s3-replication-role",  
```

Comment:
```suggestion
     ],
     "Role": "arn:aws:iam::root:role/s3-replication-role",  
```
A `,` is needed between the values to be a valid JSON snippet (applies to Create as well)